versioned_docs/version-v1.9/end-user/definition-version-control.md

```diff
@@ -0,0 +1,145 @@
+---
+title: Definition Version Control
```

Comment:
this `-v1.9` suffix version doc is generated automatically, please update the one in https://github.com/kubevela/kubevela.github.io/tree/main/docs